docs/cow-protocol/reference/core/auctions/rewards.md

```diff
@@ -41,70 +31,57 @@ From the protocol's perspective, the solution executed on chain must equal the s
+In addition to paying for gas, the winning solver might incur additional costs, such as, for example, negative slippage once a solution is settled on chain. These costs are not an explicit element of the mechanism, but they are relevant in determining the solver's optimal strategy. More precisely, per [CIP-17](https://snapshot.org/#/cow.eth/proposal/0xf9c98a2710dc72c906bbeab9b8fe169c1ed2e9af6a67776cc29b8b4eb44d0fb2), solvers are responsible for managing potential slippage incurred by the settlements they settle. This is a component that affects payouts, but can be treated completely separately, and we do so in [slippage accounting](slippage).
 
 ### Solver bidding strategies
 
-Apart from submitting their solutions, solvers must decide on a score to submit as a bid in the auction. In general, this score can be chosen freely by the solver (within the restriction $$0 < \textrm{score} < \textrm{successQuality}$$). The recommended way of submitting a score is via reporting a success probability. The score is then computed as explained below.
+After finding optimal routes, solvers must decide what solution to report. Call $$\textrm{successQuality}$$ the quality of the reported solution, which the solver can freely choose as long as it is smaller than some theoretical maximum, which we call $$\textrm{maxQuality}$$, with $$\textrm{maxQuality} - \textrm{successQuality} $$ constituting revenues to the solver.
 
-Let $$p$$ be the probability that a solution successfully executes, let $$\textrm{successCost}$$ be the costs that a solver pays if a solution successfully executes, and let $$\textrm{failCost}$$ be the costs that a solver pays if a solution does not execute successfully. For example, costs in case of success could include gas costs of the full settlement, while costs in case of failure could be small due to use of services like MEV Blocker.
-
-Ignoring capping of rewards, the winning solver's expected payoff is
+Suppose $$c_u$$ is large and can be ignored. In this case, the winning solver's expected payoff is
 
 $$
-p \cdot (\textrm{successQuality} - \textrm{referenceScore} - \textrm{successCost}) - (1 - p) \cdot (\textrm{referenceScore} + \textrm{failCost}).
+p  (\textrm{maxQuality} - \textrm{referenceQuality} - \textrm{successCost}) - (1 - p)  (\min(c_l,\textrm{referenceQuality}) + \textrm{failCost}).
 $$
 
-The optimal score is such that the solver wins if and only if it is profitable to do so, and is given by
+The key observation is that $$\textrm{successQuality}$$ doesn't affect the expected payoff in case of a win, and it only affects whether the solver wins. In particular, note that the above expression is strictly decreasing in \textrm{referenceQuality}. Hence, by choosing $$\textrm{successQuality}$$ such that
```

Comment:
```suggestion
The key observation is that $$\textrm{successQuality}$$ doesn't affect the expected payoff in case of a win, and it only affects whether the solver wins. In particular, note that the above expression is strictly decreasing in $$\textrm{referenceQuality}$$. Hence, by choosing $$\textrm{successQuality}$$ such that
```